src/rules/paddedBlocksRule.ts

```diff
@@ -0,0 +1,270 @@
+import * as ts from 'typescript';
+import * as Lint from 'tslint';
+
+import { forEachComment } from 'tsutils';
+
+const RULE_NAME = 'padded-blocks';
```

Comment:
Lets change the name to `ter-padded-blocks`. It has happened that tslint takes the names of the rule and they are slightly different from our implementation. In order to avoid surprises we can just leave the two rules living side by side, and later, if we realize that they do the same we can deprecate it if the rule from tslint does the same.